api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -55,16 +55,11 @@
 	
 	/**
 	 * @see org.openmrs.api.ConceptService#purgeConcept(org.openmrs.Concept)
-	 * @should purge concept
```

Comment:
Do you mind sharing the reason behind removing the above should purge concept?

Reply:
Hi! I have just noticed that all of the other @see did not include an @should. 
Do you think it would be better to restore the above @should purge concept? :)

Reply:
Have you read a little bit about what we use these should annotations for?

Reply:
I am very very sorry, my ignorance. I have looked it up, and understand. I will edit. Thank you :)